pkgs/development/python-modules/portolan/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildPythonPackage
+, isPy3k
+, fetchFromGitHub
+}:
+
+buildPythonPackage rec {
+  pname = "portolan";
+  version = "1.0.1";
+  disabled = !isPy3k;
```

Comment:
that does not look that promising. 
```
        'License :: OSI Approved :: GNU General Public License v3 (GPLv3)',
        'Programming Language :: Python :: 2',
        'Programming Language :: Python :: 2.7',
        'Programming Language :: Python :: 3.3',
        'Programming Language :: Python :: 3.4',
        'Topic :: Scientific/Engineering :: Atmospheric Science',
        'Intended Audience :: Science/Research',
```

but since we are removing python2 support entirely we can drop this

```suggestion
```

Reply:
Staged, will push as soon as everything is ready.